### PR TITLE
Uncomment part of the volume discounts

### PR DESCRIPTION
**Pull request by thepetk**
This PR simply uncomments the section of the volume discounts

### CODE REVIEW

File: src/content/pages/tickets.mdx [hunk 1]
```diff
@@ -89,17 +89,22 @@ Regular tickets are available until **21 June 2024**, after which Late-bird rate
...
+- get 10 business tickets of any type for the price of 8
+- get 15 business tickets of any type for the price of 11
 
-  - get 5 business tickets of any type for the price of 4
-  - get 10 business tickets of any type for the price of 8
-  - get 15 business tickets of any type for the price of 11
+Note:
 
- Note: Volume discount will not apply if you mix & match between different ticket types. -->
+- Volume discount will not apply if you mix & match between different ticket tiers (e.g. business with personal).
```
Comment (artcz) [inline src/content/pages/tickets.mdx:102]: This is correct, but maybe we can simplify by just explicitly saying it only applies only to the business tickets.

File: src/content/pages/tickets.mdx [hunk 2]
```diff
@@ -89,17 +89,22 @@ Regular tickets are available until **21 June 2024**, after which Late-bird rate
...
+- If you choose different types of tickets (e.g. Conference Only & Combined) the discount will apply to the item with the lower price.
 
+For companies interested in volume discounts, we would suggest considering being a sponsor of the event! More information can be found [here](./sponsor.mdx).
```
Comment (artcz) [inline src/content/pages/tickets.mdx:105]:
```suggestion
If you are interested in buying lots of tickets, please consider sponsoring the event! More information can be found [on the sponsors page](./sponsor.mdx).
```

File: src/content/pages/tickets.mdx [hunk 3]
```diff
@@ -89,17 +89,22 @@ Regular tickets are available until **21 June 2024**, after which Late-bird rate
...
+- get 15 business tickets of any type for the price of 11
 
-  - get 5 business tickets of any type for the price of 4
-  - get 10 business tickets of any type for the price of 8
-  - get 15 business tickets of any type for the price of 11
+Note:
 
- Note: Volume discount will not apply if you mix & match between different ticket types. -->
+- Volume discount will not apply if you mix & match between different ticket tiers (e.g. business with personal).
+- If you choose different types of tickets (e.g. Conference Only & Combined) the discount will apply to the item with the lower price.
```
Comment (artcz) [inline src/content/pages/tickets.mdx:103]:
```suggestion
- The discount will apply automatically once you have enough items in your basket.
- If you choose different types of tickets (e.g. Conference Only & Combined) the discount will apply to the item with the lower price.
```

**Review by artcz — APPROVED** [review]
LGTM w/ some minor suggestions.

**Comment by thepetk** [comment]
> LGTM w/ some minor suggestions.

Updated

**Pull request merged**